### DISCUSSION TITLE
What means "no encryption"?

### DISCUSSION THREAD

**Discussion by papanito** [twpayne/chezmoi#2251] [Q&A] [answered]
I have the following `~/.local/share/chezmoi/chezmoi.toml`

```
encryption = "gpg"
[gpg]
    symmetric = true
```
When I run
```
$ chezmoi add --encrypt ~/.mysecretfile
chezmoi: no encryption
```
Why does `--encrypt` not work and instead returns `no encryption`?

**Comment by twpayne** [comment, marked as answer]
The config file needs to be in `~/.config/chezmoi/chezmoi.toml`. Note the different directory.

**Reply by papanito** [reply]
Ups, I did not noticeed the subtle difference :-) Thanks for the quick response
